### PR TITLE
Fixes #35 and #72

### LINKED ISSUE #35: timeout for command cf
timeout for command cf

### LINKED ISSUE #72: [BUG] Warndelete
User: Ruheloses

The warn is not deleted. Warning reason: test Command: pb!warndelete @Ruheloses test

### PR DESCRIPTION
**Pull request by HGotthard**
## Requirements for Contributing

* [x] The changed and/or added Code should be documented and understandable.
* [x] The changed and/or added Code should follow the Code Conventions.

## Type of the Pull Request
* [x] Bugfix
* [x] Feature change
* [ ] Documentation change
* [ ] Something different

## Description of the Changes
Fixes #35: Adds an timeout function. Autolose after 1 day.
Fixes #72: Delete a warn by user and reason `warndelete <user> <reason>`

Adds an error message when no user is specified in the warnlist command.
Fixes a null pointer exception in the warn command when no "progenlog" channel exists.

## References
none

## Dependencies
none

### CODE REVIEW

**Review by byNoobiYT — APPROVED** [review]
Seems fine. Thanks!

File: src/main/java/de/progen_bot/commands/moderator/WarnDelete.java [hunk 1]
```diff
@@ -24,7 +27,16 @@ public void execute(CommandManager.ParsedCommandString parsedCommand, MessageRec
...
 
-        new WarnListDaoImpl().deleteWarns(event.getMember());
+        if (parsedCommand.getArgs().length <= 1) {
+            event.getChannel()
+                    .sendMessage(new EmbedBuilder().setColor(Color.red).setDescription("No reason found").build())
+                    .queue();
+            return;
+        }
+
+        String reason = String.join(" ", parsedCommand.getArgs()).replace(parsedCommand.getArgs()[0] + " ", "");
```
[the remark this answers is not in the document]
Comment (HGotthard) [reply src/main/java/de/progen_bot/commands/moderator/WarnDelete.java:37]: Yeah good idea that looks better

**Comment by byNoobiYT** [comment]
Since merging PRs, some conflicts occurred. Could you please resolve those conflicts in order the pull request to be merged?

**Comment by HGotthard** [comment]
> Since merging PRs, some conflicts occurred. Could you please resolve those conflicts in order the pull request to be merged?

The conflicts are already resolved 😊

**Comment by byNoobiYT** [comment]
Didn't notice it. Thanks!

**Pull request merged**